server.js

```diff
@@ -21,9 +23,85 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+const infoGreeting = {
+  Info: 
+  "Gallery on the Go API",
+
+  Endpoints: [
+    {
+   "/prints": "lists all of the prints",
+   "/prints/year/{year}": "find prints per year created",
+   "/prints/title/{title}": "find prints per title",
+   "/prints/price/{price}": "see price",
+   "/prints/media/{media}": "see what type of media was used"
+    }
+  ]
+
+}
+  res.send(infoGreeting);
 });
 
+app.get("/prints", (req, res) => {
+  const { title, year, media, price, size, url } = req.query;
+  
+  const allPrints = prints;
+
+  if (title) {
+    allPrints = allPrints.filter(print => print.title.toLowerCase() === title.toLowerCase())
+  }
+
+  // if (media) {
+  //   allPrints = allPrints.filter(print => print.media.toLowerCase() === media.toLowerCase())
+  // }
+
+  res.status(200).json({
+    data: prints,
+    success: true,
+  });
+});
+
+app.get("/prints/year/:year", (req, res) => {
+  const { year } = req.params;
+
+  const printByYear = prints.filter(print => print.year === year);
+  console.log('print made year:', printByYear)
+
+  if (!printByYear) {
+    res.status(404).json({
```

Comment:
I'm unable to enter this if statement and get the 404 status when I try putting years that don't exist in your data. It still gives me the success and 200 status. (I have the same issue so I don't currently have the fix but I'll look into it when I have more time).